subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/report/Report.kt

```diff
@@ -150,6 +158,18 @@ interface Report {
             }
         }
 
+        override fun getTests(): Try<List<SimpleRunTest>> {
+            return reportsApi.getTestsForRunId(reportCoordinates)
+        }
+
+        override fun markAsSuccessful(testRunId: String, author: String, comment: String): Try<Unit> {
+            return reportsApi.markAsSuccessful(testRunId, author, comment)
```

Comment:
why do you think it's a good idea to write these proxies? 
They have no real value, but for possible Allure implementation; i didn't saw such intent in this PR, so it's unclear.
If it's for allure, we definitely need to move this code away from this module